src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -51,12 +53,13 @@
     private final String folder;
     private final String customizationSpec;
     private final boolean powerOn;
+    private final Integer timeoutInSeconds;
```

Comment:
I think that, unless we want to store a null/empty timeoutInSeconds field, this should be "int".
Pros/Cons...
If we have this field as an "int" then, if the user opens and presses "Save" on the config without setting the value, this field will be added with the default value.
If we have this field as an "Integer" then the field can be persisted as having no value set, meaning that the persisted data is basically saying "we want the default timeout to be used, whatever that is".  In this case, the code should be modified to do the "if this is null, use the default value" at execution-time instead of at construction time.

Personally I think it'd be simpler to have this as an int, and have the get method also return an int.
(if you want it as an Integer then you'll have to delay the "if null..." stuff until the perform(...) method.